### PR TITLE
Task/MON-7

### PR DESCRIPTION
**Pull request by aa-kashk**
Updated all dependencies in pom.xml

### CODE REVIEW

File: monitor/pom.xml [hunk 1]
```diff
@@ -203,28 +203,28 @@
...
-			<version>2.6.1</version>
+			<version>3.4.2</version>
 		</dependency>
 
 		<!-- JPA Provider (Hibernate) -->
 		<dependency>
 			<groupId>org.hibernate</groupId>
 			<artifactId>hibernate-entitymanager</artifactId>
-			<version>5.2.9.Final</version>
+			<version>6.0.0.Alpha4</version>
```
Comment (jdamerow) [inline monitor/pom.xml]: let's stick to the latest stable version, 5.4.12.Final I believe

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by aa-kashk** [comment]
Updated the hibernate-entitymanager version to 5.4.12.Final

File: monitor/pom.xml [hunk 2]
```diff
@@ -8,10 +8,10 @@
 	<packaging>war</packaging>
 
 	<properties>
-		<org.slf4j-version>1.7.5</org.slf4j-version>
-		<org.springframework-version>5.0.0.RELEASE</org.springframework-version>
-		<spring-security-version>4.2.3.RELEASE</spring-security-version>
-		<admin.password>admin</admin.password>
+		<org.slf4j-version>2.0.0-alpha1</org.slf4j-version>
```
Comment (jdamerow) [inline monitor/pom.xml]: let's not use unstable versions

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by aa-kashk** [comment]
Updated slf4j version to 1.7.30

File: monitor/src/main/webapp/WEB-INF/tiles/skeleton.jsp [hunk 3]
```diff
@@ -73,7 +73,7 @@
          
         </nav>
         
-        <h1><a class="appName" href="<c:url value="/" />">Monitor</a> <span style="margin-left: 15px;" class="small">How are your sites are doing?</span></h1>
+        <h1><a class="appName" href="<c:url value="/" />">Monitor</a> <span style="margin-left: 15px;" class="small">How are your sites doing?</span></h1>
```
Comment (jdamerow) [inline monitor/src/main/webapp/WEB-INF/tiles/skeleton.jsp:76]: Alright, I'll test it with that just so we can get this done with, but PLEASE from now on keep your changes contained to their own branches!

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
test this please

**Comment by jdamerow** [comment]
test this please

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
trying to login, throws an exception:
```
java.lang.IllegalArgumentException: There is no PasswordEncoder mapped for the id "null"
	org.springframework.security.crypto.password.DelegatingPasswordEncoder$UnmappedIdPasswordEncoder.matches(DelegatingPasswordEncoder.java:250)
	org.springframework.security.crypto.password.DelegatingPasswordEncoder.matches(DelegatingPasswordEncoder.java:198)
	org.springframework.security.authentication.dao.DaoAuthenticationProvider.additionalAuthenticationChecks(DaoAuthenticationProvider.java:90)
```

**Comment by jdamerow** [comment]
I assume it has something to do with adding the {noop} prefix to the admin password. When deployed to staging/production this is an encoded password.

**Comment by aa-kashk** [comment]
The prefix when using an encoded password is {bcrypt} but this is preventing login when I try to run it on my local env.

File: monitor/src/main/webapp/WEB-INF/tiles/skeleton.jsp (hunk 3 above)
Comment (aa-kashk) [reply monitor/src/main/webapp/WEB-INF/tiles/skeleton.jsp:76]: I'm sorry about this. I did a git pull incorrectly and it merged mon3 with this branch.

**Comment by jdamerow** [comment]
To keep these two things separate, let's make the prefix it's own maven property, set it to bcrypt by default and just use an encrypted admin password. Ideally, I don't want to change the build configuration just for this and it makes more sense to keep the password separate anyways.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Pull request merged**